app/views/includes/phase_banner.njk

```diff
@@ -4,7 +4,26 @@
     <h4 class="service-info--name govuk-heading-s">
       {{currentService.name}}
       {% if currentGatewayAccount %}
-        <strong class="environment-tag environment-tag-{{currentGatewayAccount.type}}">{% if currentGatewayAccount.full_type === 'Sandbox test' %}Test{% else %}{{currentGatewayAccount.full_type}}{% endif %}</strong>
+        {% set stripeAccountIsSetup = (currentGatewayAccount.accountSetupDetails.bankAccount and currentGatewayAccount.accountSetupDetails.responsiblePerson and currentGatewayAccount.accountSetupDetails.vatNumberCompanyNumber) %}
+        {% set tagClass %}
+          {% if currentGatewayAccount.type === "test" %}
+            govuk-tag--grey
+          {% elseif currentGatewayAccount.payment_provider === 'stripe' and not stripeAccountIsSetup %}
+            govuk-tag--blue
+          {% else %}
+
+          {% endif %}
+        {% endset %}
+        {% set tagText %}
+          {% if currentGatewayAccount.full_type === "Sandbox test" %}
+            Test
+          {% elseif currentGatewayAccount.payment_provider === 'stripe' and not stripeAccountIsSetup %}
+            {{ currentGatewayAccount.full_type }} - information needed
```

Comment:
what are the different values of `full_type`?

Reply:
It's very curious, this just moves the logic that's already there.

I think it all needs a rethink when we're looking at cleaning up/ potentially restructuring the admin tool: 
https://github.com/alphagov/pay-selfservice/blob/a3e26b0fc1a08f628c8aa6ae814e5ffdcc91beaa/app/utils/display_converter.js#L78-L80

Reply:
Dh pretty curious. I guess it's going to show "Worldpay test" in some cases?
Do we want the tag to be always in full caps now?

Could we maybe move all this display formatting into the template to make it clearer what's going on?

Reply:
Yeah I think that's the idea 'Test' is used for sandbox accounts but if it's a test account with an integration it's more specific. 

I think all caps looks much better and is in-line with the design system but would be happy to discuss it.

Yeah totally agreed with moving the formatting into the template, we'll just need to dig into how many different downstream things use the `full_type` attribute.